### ISSUE TITLE
Error on Pusher v5

### ISSUE THREAD

**Issue by taylorotwell** [pusher/pusher-http-php#283]
Hi - I'm the maintainer of Laravel.

When trying to use version 5 (I am adding support for this version to Laravel) of this library I get the following error:

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/463230/108733018-b5266780-74f3-11eb-85a7-5b6a4c43e0d5.png">

It appears to be in Pusher vendor code?

**Comment by taylorotwell** [comment]
Also the message **does** broadcast successfully. I can receive it on the client. But Pusher's internal handling of the response is apparently breaking?

**Comment by ben-pusher** [comment]
What is the response you receive? 
I have tested triggering an event as follows: 
`  $data['message'] = 'hello world';
   $pusher->trigger(['my-channel', 'my-channel-2'], 'my-event', $data);`

I don't receive an error, the response I receive is: 
```
(
    [body] => {}
    [status] => 200
)
```

I'm not using Laravel, just the pusher-http-php library.

**Comment by taylorotwell** [comment]
Within `Pusher.php` line 536... $result is an empty object.

<img width="302" alt="image" src="https://user-images.githubusercontent.com/463230/108745108-54515c00-7500-11eb-897f-63ab2e35f2cb.png">

**Comment by taylorotwell** [comment]
```php
        $result = json_decode($response['body']);

        if ($result->channels) {
            $result->channels = get_object_vars($result->channels);
        }
```

Within this code in `Pusher.php`, it looks like you don't expect `channels` to always be there - maybe that should be changed to `if (isset($result->channels))`?

**Comment by Koozza** [comment]
@taylorotwell I already created a PR for this when I also created the PR for laravel :)

https://github.com/pusher/pusher-http-php/pull/281
https://github.com/laravel/framework/pull/36344

**Comment by WillSewell** [comment]
Thanks for reporting this issue.

The reason this was missed is it is a PHP "notice", which is not considered an error in the tests currently.

I assume that in Laravel these notices are automatically translated into ErrorExceptions.

**Comment by WillSewell** [comment]
Fixed in https://github.com/pusher/pusher-http-php/pull/281, which has been released in v5.0.1.
